docs/structure.md

```diff
@@ -39,11 +45,11 @@ silently exits with status code 0. In case the test failed an error is printed
 and the command exits with status code 1.
 
 ```
-$ awscnfm cl001 ac001 execute
+$ awscnfm cl001 ac007 execute
```

Comment:
I changed this due to the implications of the first action which I described in the docs above.